app/search.py

```diff
@@ -15,12 +15,32 @@ def open_browser(urls):
 
 def search_my_term(term, count):
     return [url for url in search(term, stop=count)]
+def search_rem(term,s,count):
+    return [url for url in search(term,start=s,stop=count)]
 
+def ignore():
+    with open("/home/olagammal/Desktop/search-assist-master/ignore.txt") as f:
+        return f.read().splitlines()
 
+url_list=[]
 term = sys.argv[1]
 count = int(sys.argv[2])
 
 print (term, count)
 
+ignore_list=ignore()
+final=0
 urls = search_my_term(term, count)
+for url in urls:
+    if url not in ignore_list:
```

Comment:
The logic here expects full url but in ignore.txt we will only save the domain name for eg.,

```
udemy.com
edureka.com
```
The program should ignore any link in that URL